libs/ledger/include/ledger/consensus/consensus.hpp

```diff
@@ -82,6 +82,7 @@ class Consensus final : public ConsensusInterface
   Consensus &operator=(Consensus const &) = delete;
   Consensus &operator=(Consensus &&) = delete;
 
+  // TODO (nobody): the only reason this function is public is it's used in a single POS test.
```

Comment:
Can you create a JIRA issue to track this and update the code

Reply:
LGDR-698